packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -151,6 +159,7 @@ export class VirtualGateway extends VirtualGatewayBase {
   public readonly mesh: IMesh;
 
   protected readonly listeners = new Array<VirtualGatewayListenerConfig>();
+  private readonly backendDefaults: CfnVirtualGateway.VirtualGatewayBackendDefaultsProperty;
```

Comment:
Why does this need to be a field? It's only set and read in the constructor of the class?

Reply:
I have updated the PR to use an array here. This gives me an option set `omitEmptyArray` true and this would remove backendDefault if not defined.